server/test/unit/server/webservices/views/test_repo_groups.py

```diff
@@ -0,0 +1,506 @@
+import json
+import unittest
+
+import mock
+
+from .base import (assert_auth_CREATE, assert_auth_DELETE, assert_auth_EXECUTE, assert_auth_READ,
+                   assert_auth_UPDATE)
+from pulp.common import error_codes
+from pulp.server import exceptions as pulp_exceptions
+from pulp.server.webservices.views.repo_groups import (
+    RepoGroupAssociateView, RepoGroupDistributorResourceView, RepoGroupDistributorsView,
+    RepoGroupPublishView, RepoGroupResourceView, RepoGroupsView, RepoGroupUnassociateView
+)
+
+
+class TestRepoGroupsView(unittest.TestCase):
+    """
+    Tests for RepoGroupsView.
+    """
+
+    @mock.patch('pulp.server.webservices.controllers.decorators._verify_auth',
+                new=assert_auth_READ())
+    @mock.patch('pulp.server.webservices.views.repo_groups.reverse')
+    @mock.patch('pulp.server.webservices.views.repo_groups.RepoGroupModel')
```

Comment:
I think you can mock directly here 'pulp.server.webservices.views.repo_groups.RepoGroupModel.get_collection'